presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1107,6 +1108,11 @@ public SystemSessionProperties(
                         "Enable query optimization with materialized view",
                         featuresConfig.isQueryOptimizationWithMaterializedViewEnabled(),
                         true),
+                booleanProperty(
+                        ENABLE_DISTRIBUTED_TRACING,
+                        "Enable distributed tracing of the query",
+                        false,
```

Comment:
let's make a new config and set it here. Something like `tracerConfig.isTracerEnabled()`

Reply:
I think we may want to discuss this topic again in a call. I don't quite understand why we need 2 settings to tell the system to enable/disable tracing.